miprometheus/models/model.py

```diff
@@ -317,38 +317,35 @@ def plot(self, data_dict, predictions, sample=0):
 
         """
 
-    def save(self, model_dir, stats):
+    def save(self, model_dir, training_status, stats):
```

Comment:
So you are adding a new argument to this method, which means that where it is called (with `super()` for instance), it needs to be modified as well...

Reply:
LGTM has caught that the call to this function in `MAES.save()` does not have 3 arguments, so it will break

Reply:
I know, I saw that as well, fixing...;)